src/modules/Add_FungibleTokenSwitchboard_Resource/description/description.js

```diff
@@ -0,0 +1,17 @@
+export const name = 'FungibleTokenSwitchboard';
+
+export const description =
+  'This explains the FungibleTokenSwitchboard resource that can be created in order for you to receive payments in different fungible tokens .';
+
+export const playgroundLink =
+  'https://play.onflow.org/8b168390-9712-4578-a437-d15125cf18d9';
```

Comment:
This links to a ExampleToken tutorial on playground, I don't see anything related to the switchboard here

Reply:
I've removed it, I am not sure how to add a working playground example yet, would need to have a look at some point, for now, I've just removed the link.

Reply:
It's quite simple actually, just get into https://play.flow.com/ and once you click `Save` it will create the URL for your new project.

Reply:
Thanks. Added on [c70090e](https://github.com/onflow/cadence-cookbook/pull/7/commits/c70090ed7698d8f2c13ade2a7fb42d1dbeea709c).